inventory/byo/hosts.origin.example

```diff
@@ -571,10 +571,18 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # network blocks should be private and should not conflict with network blocks
 # in your infrastructure that pods may require access to. Can not be changed
 # after deployment.
+#
+# WARNING : Do not pick subnets that overlap with the default Docker bridge subnet of
+# 172.16.0.0/16.  Your installation will fail and/or your configuration change will
```

Comment:
That contradicts what https://docs.openshift.com/container-platform/3.4/install_config/install/advanced_install.html#configuring-host-variables says, which is correct?

```
Defaults to 172.30.0.0/16, and cannot be re-configured after deployment. If changing
from the default, avoid 172.16.0.0/16, which the docker0 network bridge uses by
default, or modify the docker0 network.
```